src/models/Factor.js

```diff
@@ -212,7 +212,11 @@ function (Okta, Q, factorUtil, Util, Errors, BaseLoginModel) {
         return {'password': Okta.loc('error.password.required')};
       }
     },
-
+    addPasscode: function () {
```

Comment:
nit: `needsPasscode`, otherwise I will assume the functions adds the passcode itself

Reply:
fixed